patches/buildroot/0001-disable-shared-for-host-builds-of-gmp-isl-mpc-mpfr.patch

```diff
@@ -1,7 +1,7 @@
-From 9e6d1c176a007d6751e3acd02f96becee48ef361 Mon Sep 17 00:00:00 2001
+From b90b648d30fe4b6e6554a04b9d3f7d93aebdf6c7 Mon Sep 17 00:00:00 2001
 From: Ben Cressey <bcressey@amazon.com>
 Date: Sun, 6 Aug 2017 23:15:02 +0000
-Subject: [PATCH] disable shared for host builds of gmp, isl, mpc, mpfr
+Subject: [PATCH 1/6] disable shared for host builds of gmp, isl, mpc, mpfr
```

Comment:
You should remove the patch numbers with `--no-numbered`

Reply:
Done